projects/isy-angular-widgets/src/lib/validation/validation.spec.ts

```diff
@@ -103,10 +111,16 @@ describe('Unit Test: Validation', () => {
     validInputs.forEach((input) => {
       it(`should return null for '${input}'`, () => {
         control.setValue(input);
-        const errors = Validation.validUnspecifiedISODate(control);
+        const errors = Validation.validUnspecifiedISODate(control, true);
         expect(errors).toBeNull();
       });
     });
+
+    it('should return an error for the input 0000-00-00 when the zero date format is not allowed', () => {
+      control.setValue('0000-00-00');
+      const errors = Validation.validUnspecifiedISODate(control, false);
+      expect(errors).toEqual({[errorKey]: true});
```

Comment:
Was wird in diesem Fall für eine Fehlermeldung angezeigt? In dem Fall ist vermutlich gemeint das das Datum ungültig ist oder?